tools/integration_tests/run_e2e_tests.sh

```diff
@@ -271,6 +294,11 @@ function main(){
 
   set +e
 
+  # Run test in TPC environment
```

Comment:
Please correct the comment: `Run tpc test and exist in case RUN_TEST_ON_TPC_ENDPOINT is true.`
Anyhow if we can write exit within this block, that will improve readability.